### PR TITLE
Update documentation

### PR DESCRIPTION
**Pull request by charly-palencia**
- Include Mason docs
- Include custom recursive fonts
- Incluce blank docs documents

### CODE REVIEW

**Comment by charly-palencia** [comment]
@EdgarOrtegaRamirez  merge with squash for this PR please

File: README.md [hunk 1]
```diff
@@ -46,6 +46,17 @@ Open `nvim` and enter the following:
 :checkhealth
 ```
 
+## Install LSP (Language Service Protocol)
```
Comment (EdgarOrtegaRamirez) [inline README.md]:
```suggestion
## Install LSP (Language Server Protocol) Servers
```

File: README.md [hunk 2]
```diff
@@ -46,6 +46,17 @@ Open `nvim` and enter the following:
...
+
+Facilitates features like go-to-definition, find-references, hover, completion, rename, format, refactor, etc., using semantic whole-project analysis. 
+
+Neovim dotfile uses [Mason.nvim](https://github.com/williamboman/mason.nvim) to easly manage LSP Servers. Here's [the complete LSP list](https://github.com/williamboman/mason-lspconfig.nvim#available-lsp-servers). To install LSP run `:Mason`.
```
Comment (EdgarOrtegaRamirez) [inline README.md]:
```suggestion
Neovim dotfile uses [Mason.nvim](https://github.com/williamboman/mason.nvim) to easly manage LSP Servers and other tools. Here's [the complete LSP list](https://github.com/williamboman/mason-lspconfig.nvim#available-lsp-servers). Run `:Mason` to see all of the available packages to install.
```

File: README.md [hunk 3]
```diff
@@ -46,6 +46,17 @@ Open `nvim` and enter the following:
...
+
+![mason](https://user-images.githubusercontent.com/6705160/177617680-d62caf26-f253-4ace-ab57-4b590595adca.png)
+
+
+## Copy & Pase Support
+
 You'll probably notice you don't have support for copy/paste also that python and node haven't been setup
```
Comment (EdgarOrtegaRamirez) [inline README.md]:
```suggestion
You'll probably notice you don't have support for copy/paste, also that Python and Node.js haven't been setup.
```

File: README.md [hunk 4]
```diff
@@ -86,16 +97,18 @@ Next we need to install python support (node is optional)
 
 - [codicon](https://github.com/microsoft/vscode-codicons/raw/main/dist/codicon.ttf)
 - [An emoji font](https://github.com/googlefonts/noto-emoji/blob/main/fonts/NotoColorEmoji.ttf)
+- For custom [Recursive fonts](https://www.recursive.design/) with [Nerdfont icons](https://www.nerdfonts.com/), go to the `assets/fonts` folder and download the
```
Comment (EdgarOrtegaRamirez) [inline README.md]:
```suggestion
- A patched version of [Recursive font](https://www.recursive.design/) with [Nerdfont icons](https://www.nerdfonts.com/) added. Go to the `assets/fonts` folder and download the
```

File: docs/installed_plugins.md [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+| nvim-web-devicons       | A lua fork of vim-devicons. This plugin provides the same icons as well as colors for each icon.                                                                                              | [kyazdani42/nvim-web-devicons](https://github.com/kyazdani42/nvim-web-devicons)                               |
+| Trouble                 | A pretty list for showing diagnostics, references, telescope results, quickfix and location lists to help you solve all the trouble your code is causing.                                     | [folke/trouble.nvim](https://github.com/folke/trouble.nvim)                                                   |
+| vim-tree.lua            | A File Explorer For Neovim Written In Lua                                                                                                                                                     | [kyazdani42/nvim-tree.lua](https://github.com/kyazdani42/nvim-tree.lua)                                       |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| nvim-tree.lua            | A File Explorer For Neovim Written In Lua                                                                                                                                                     | [kyazdani42/nvim-tree.lua](https://github.com/kyazdani42/nvim-tree.lua)                                       |
```

File: docs/installed_plugins.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+| Fidget                  | Standalone UI for nvim-lsp progress. Eye candy for the impatient.                                                                                                                             | [j-hui/fidget.nvim](https://github.com/j-hui/fidget.nvim)                                                     |
+| Toggle-lsp-diagnostics  | A Neovim plugin for toggling the LSP diagnostics.  Turn all diagnostics on/off or turn on/off individual features of diagnostics                                                              | [WhoIsSethDaniel/toggle-lsp-diagnostics.nvim](https://github.com/WhoIsSethDaniel/toggle-lsp-diagnostics.nvim) |
+| Catppuccino             | Catppuccino.nvim is a NeoVim plugin that provides multiple colorschemes based on the Catppuccino color palette but varying their properties                                                   | [catppuccin/nvim](https://github.com/catppuccin/nvim)                                                         |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| Catppuccin             | NeoVim plugin that provides multiple colorschemes based on the Catppuccin color palette but varying their properties                                                   | [catppuccin/nvim](https://github.com/catppuccin/nvim)                                                         |
```

File: docs/installed_plugins.md [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+
+
+## LSP Plugins
+
+| Name                        | Description                                                                                                                                                                       | Link                                                                                          |
+|-----------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|
+| null-ls.nvim                | Use Neovim as a language server to inject LSP diagnostics, code actions, and more via Lua.                                                                                        | [jose-elias-alvarez/null-ls.nvim](https://github.com/jose-elias-alvarez/null-ls.nvim          |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| null-ls.nvim                | Use Neovim as a language server to inject LSP diagnostics, code actions, and more via Lua.                                                                                        | [jose-elias-alvarez/null-ls.nvim](https://github.com/jose-elias-alvarez/null-ls.nvim)          |
```

File: docs/installed_plugins.md [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+| Lsp-colors.nvim             | Automatically creates missing LSP diagnostics highlight groups for color schemes that don't yet support the Neovim 0.5 builtin lsp client.                                        | [folke/lsp-colors.nvim](https://github.com/folke/lsp-colors.nvim)                             |
+| nvim-lspconfig              | The Nvim LSP client does not live here. This is only a collection of LSP configs.                                                                                                 | [neovim/nvim-lspconfig](https://github.com/neovim/nvim-lspconfig)                             |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| nvim-lspconfig              | Collection of LSP Configs for the Nvim LSP client.                                                                                                 | [neovim/nvim-lspconfig](https://github.com/neovim/nvim-lspconfig)                             |
```

File: docs/installed_plugins.md [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+| Lspkind                     | This tiny plugin adds vscode-like pictograms to neovim built-in lsp                                                                                                               | [onsails/lspkind.nvim](https://github.com/onsails/lspkind.nvim)                               |
+| nvim-cmp                    | A completion engine plugin for neovim written in Lua. Completion sources are installed from external repositories and "sourced".                                                  | [hrsh7th/nvim-cmp](https://github.com/hrsh7th/nvim-cmp)                                       |
+|                             |                                                                                                                                                                                   |                                                                                               |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]: remove empty line
```suggestion                            
```

File: docs/installed_plugins.md [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+
+## Autocompletion
+
+| Name                        | Description                                                                                                                                                                       | Link                                                                                          |
+|-----------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|
+| null-ls.nvim                | Use Neovim as a language server to inject LSP diagnostics, code actions, and more via Lua.                                                                                        |  [jose-elias-alvarez/null-ls.nvim](https://github.com/jose-elias-alvarez/null-ls.nvim)          |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]: already added in the LSP section
```suggestion
```

File: docs/installed_plugins.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+| Lsp-colors.nvim             | Automatically creates missing LSP diagnostics highlight groups for color schemes that don't yet support the Neovim 0.5 builtin lsp client.                                        | [folke/lsp-colors.nvim](https://github.com/folke/lsp-colors.nvim)                             |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]: already added in the LSP section
```suggestion
```

File: docs/installed_plugins.md [hunk 12]
```diff
@@ -0,0 +1,103 @@
...
+| nvim-lspconfig              | The Nvim LSP client does not live here. This is only a collection of LSP configs.                                                                                                 | [neovim/nvim-lspconfig](https://github.com/neovim/nvim-lspconfig)                             |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]: already added in the LSP section
```suggestion
```

File: docs/installed_plugins.md [hunk 13]
```diff
@@ -0,0 +1,103 @@
...
+| Mason                       | Portable package manager for Neovim that runs everywhere Neovim runs.                                                                                                             | [williamboman/mason.nvim](https://github.com/williamboman/mason.nvim)                         |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]: already added in the LSP section
```suggestion
```

File: docs/installed_plugins.md [hunk 14]
```diff
@@ -0,0 +1,103 @@
...
+| mason-lspconfig             | mason-lspconfig bridges mason.nvim with the lspconfig plugin - making it easier to use both plugins together.                                                                     | [williamboman/mason-lspconfig.nvim](https://github.com/williamboman/mason-lspconfig.nvim)     |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]: already added in the LSP section
```suggestion
```

File: docs/installed_plugins.md [hunk 15]
```diff
@@ -0,0 +1,103 @@
...
+| lsp_signature               | This nvim plugin is made for completion plugins that do not support signature help. Need neovim-0.6.1+ and enable nvim-lsp. (check neovim-0.5 branch for earlier version support) | [ray-x/lsp_signature.nvim](https://github.com/ray-x/lsp_signature.nvim)                       |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]: already added in the LSP section
```suggestion
```

File: docs/installed_plugins.md [hunk 16]
```diff
@@ -0,0 +1,103 @@
...
+| cmp-nvim-lsp                | nvim-cmp source for neovim's built-in language server client.                                                                                                                     | [hrsh7th/cmp-nvim-lsp](https://github.com/hrsh7th/cmp-nvim-lsp)                               |
+| cmp-nvim-lsp-signature-help | nvim-cmp source for displaying function signatures with the current parameter emphasized                                                                                          | [hrsh7th/cmp-nvim-lsp-signature-help](https://github.com/hrsh7th/cmp-nvim-lsp-signature-help) |
+| Lspsaga                     | A light-weight lsp plugin based on neovim's built-in lsp with a highly performant UI.                                                                                             | [glepnir/lspsaga.nvim](https://github.com/glepnir/lspsaga.nvim)                               |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]: already added in the LSP section
```suggestion
```

File: docs/installed_plugins.md [hunk 17]
```diff
@@ -0,0 +1,103 @@
...
+| Lspkind                     | This tiny plugin adds vscode-like pictograms to neovim built-in lsp                                                                                                               | [onsails/lspkind.nvim](https://github.com/onsails/lspkind.nvim)                               |
+| nvim-cmp                    | A completion engine plugin for neovim written in Lua. Completion sources are installed from external repositories and "sourced".                                                  | [hrsh7th/nvim-cmp](https://github.com/hrsh7th/nvim-cmp)                                       |
+|                             |                                                                                                                                                                                   |                                                                                               |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]: remove empty line
```suggestion
```

File: docs/installed_plugins.md [hunk 18]
```diff
@@ -0,0 +1,103 @@
...
+| Filetype              | Easily speed up your neovim startup time!                                                                                                                                                       | [nathom/filetype.nvim](https://github.com/nathom/filetype.nvim)                                         |
+| Popup                 | An implementation of the Popup API from vim in Neovim. Hope to upstream when complete                                                                                                           | [nvim-lua/popup.nvim](https://github.com/nvim-lua/popup.nvim)                                           |
+| plenary               | All the lua functions I don't want to write twice.                                                                                                                                              | [nvim-lua/plenary.nvim](https://github.com/nvim-lua/plenary.nvim)                                       |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| plenary               | Set of common Lua functions used internally by other plugins                                                                                                                                              | [nvim-lua/plenary.nvim](https://github.com/nvim-lua/plenary.nvim)                                       |
```

File: docs/installed_plugins.md [hunk 19]
```diff
@@ -0,0 +1,103 @@
...
+| indent-blankline      | This plugin adds indentation guides to all lines (including empty lines).                                                                                                                       | [lukas-reineke/indent-blankline.nvim](https://github.com/lukas-reineke/indent-blankline.nvim)           |
+| gitsigns              | Super fast git decorations implemented purely in lua/teal.                                                                                                                                      | [lewis6991/gitsigns.nvim](https://github.com/lewis6991/gitsigns.nvim)                                   |
+| LuaSnip               | Dynamic function comments                                                                                                                                                                       | [L3MON4D3/LuaSnip](https://github.com/L3MON4D3/LuaSnip)                                                 |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| LuaSnip               | Snippet Engine for Neovim written in Lua.                                                                                                                                                                       | [L3MON4D3/LuaSnip](https://github.com/L3MON4D3/LuaSnip)                                                 |
```

File: docs/installed_plugins.md [hunk 20]
```diff
@@ -0,0 +1,103 @@
...
+| telescope-fzy-native  | FZY style sorter that is compiled                                                                                                                                                               | [nvim-telescope/telescope-fzy-native.nvim](https://github.com/nvim-telescope/telescope-fzy-native.nvim) |
+| nvim-pqf              | Prettier quickfix/location list windows for NeoVim                                                                                                                                              | [yorickpeterse/nvim-pqf](https://gitlab.com/yorickpeterse/nvim-pqf)                                     |
+| luatab                | ust a lua rewrite of the tabline render function                                                                                                                                                | [alvarosevilla95/luatab.nvim](https://github.com/alvarosevilla95/luatab.nvim)                           |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| luatab                |  Tabline lua plugin for neovim                                                                                                                                                 | [alvarosevilla95/luatab.nvim](https://github.com/alvarosevilla95/luatab.nvim)                           |
```

File: docs/installed_plugins.md [hunk 21]
```diff
@@ -0,0 +1,103 @@
...
+| telescope             | Gaze deeply into unknown regions using the power of the moon.                                                                                                                                   | [nvim-telescope/telescope.nvim](https://github.com/nvim-telescope/telescope.nvim)                       |
+| telescope-ui-select   | It sets vim.ui.select to telescope. That means for example that neovim core stuff can fill the telescope picker. Example would be lua vim.lsp.buf.code_action().                                | [nvim-telescope/telescope-ui-select.nvim](https://github.com/nvim-telescope/telescope-ui-select.nvim)   |
+| telescope-fzy-native  | FZY style sorter that is compiled                                                                                                                                                               | [nvim-telescope/telescope-fzy-native.nvim](https://github.com/nvim-telescope/telescope-fzy-native.nvim) |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| telescope-fzy-native  | FZY style sorter                                                                                                                                                               | [nvim-telescope/telescope-fzy-native.nvim](https://github.com/nvim-telescope/telescope-fzy-native.nvim) |
```

File: docs/installed_plugins.md [hunk 22]
```diff
@@ -0,0 +1,103 @@
...
+| wilder                | Vim command tools to Automatically provides suggestions as you type                                                                                                                             | [gelguy/wilder.nvim](https://github.com/gelguy/wilder.nvim)                                             |
+| nvim-notify           | A fancy, configurable, notification manager for NeoVim                                                                                                                                          | [rcarriga/nvim-notify](https://github.com/rcarriga/nvim-notify)                                         |
+| stabilize             | Neovim plugin to stabilize buffer content on window open/close events. There is a currently outstanding                                                                                         | [luukvbaal/stabilize.nvim](https://github.com/luukvbaal/stabilize.nvim)                                 |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| stabilize             | Neovim plugin to stabilize buffer content on window open/close events.                                                                                          | [luukvbaal/stabilize.nvim](https://github.com/luukvbaal/stabilize.nvim)                                 |
```

File: docs/installed_plugins.md [hunk 23]
```diff
@@ -0,0 +1,103 @@
...
+| numb                  | numb.nvim is a Neovim plugin that peeks lines of the buffer in non-obtrusive way.                                                                                                               | [nacro90/numb.nvim](https://github.com/nacro90/numb.nvim)                                               |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| numb                  | Neovim plugin that peeks lines of the buffer in non-obtrusive way.                                                                                                               | [nacro90/numb.nvim](https://github.com/nacro90/numb.nvim)                                               |
```

File: docs/installed_plugins.md [hunk 24]
```diff
@@ -0,0 +1,103 @@
...
+| dressing              | It is a goal to match and not extend the core Neovim API.                                                                                                                                       | [stevearc/dressing.nvim](https://github.com/stevearc/dressing.nvim)                                     |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| dressing              |  Neovim plugin to improve the default vim.ui interfaces                                                                                                                                       | [stevearc/dressing.nvim](https://github.com/stevearc/dressing.nvim)                                     |
```

File: docs/installed_plugins.md [hunk 25]
```diff
@@ -0,0 +1,103 @@
...
+| markdown-preview      | Preview markdown on your modern browser with synchronised scrolling and flexible configuration                                                                                                  | [iamcco/markdown-preview.nvim](https://github.com/iamcco/markdown-preview.nvim)                         |
+| focus                 | Auto-Resizing Focused Splits/Windows for Neovim                                                                                                                                                 | [beauwilliams/focus.nvim](https://github.com/beauwilliams/focus.nvim)                                   |
+| nvim-hlslens          | nvim-hlslens helps you better glance at matched information, seamlessly jump between matched instan                                                                                             | [kevinhwang91/nvim-hlslens](https://github.com/kevinhwang91/nvim-hlslens)                               |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| nvim-hlslens          |  Hlsearch Lens for Neovim                                                                                              | [kevinhwang91/nvim-hlslens](https://github.com/kevinhwang91/nvim-hlslens)                               |
```

File: docs/installed_plugins.md [hunk 26]
```diff
@@ -0,0 +1,103 @@
...
+| git-conflict          | git-conflict.nvim                                                                                                                                                                               | [akinsho/git-conflict.nvim](https://github.com/akinsho/git-conflict.nvim)                               |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| git-conflict          |  A plugin to visualise and resolve merge conflicts in neovim                                                                                                                                                                                | [akinsho/git-conflict.nvim](https://github.com/akinsho/git-conflict.nvim)                               |
```

File: docs/installed_plugins.md [hunk 27]
```diff
@@ -0,0 +1,103 @@
...
+| mini                  | Library of 20+ independent Lua modules improving overall experience with minimal effort. They all share same configuration approaches and general design principles.                            | [echasnovski/mini.nvim](https://github.com/echasnovski/mini.nvim)                                       |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| mini                  | Library of 20+ independent Lua modules improving overall experience with minimal effort.                            | [echasnovski/mini.nvim](https://github.com/echasnovski/mini.nvim)                                       |
```

File: docs/installed_plugins.md [hunk 28]
```diff
@@ -0,0 +1,103 @@
...
+| nvim-lightbulb        | The plugin shows a lightbulb in the sign column whenever a textDocument/codeAction is available at the current cursor position.                                                                 | [kosayoda/nvim-lightbulb](https://github.com/kosayoda/nvim-lightbulb)                                   |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]:
```suggestion
| nvim-lightbulb        | shows a lightbulb in the sign column whenever a textDocument/codeAction is available at the current cursor position.                                                                 | [kosayoda/nvim-lightbulb](https://github.com/kosayoda/nvim-lightbulb)                                   |
```

File: docs/installed_plugins.md [hunk 29]
```diff
@@ -0,0 +1,103 @@
...
+| vim-repeat            | repeated the last native command inside that map                                                                                                                                                | [tpope/vim-repeat](https://github.com/tpope/vim-repeat)                                                 |
+| vim-matchup           | match-up is a plugin that lets you highlight, navigate, and operate on sets of matching text. It extends vim's % key to language-specific words instead of just single characters               | [andymass/vim-matchup](https://github.com/andymass/vim-matchup)                                         |
+|                       |                                                                                                                                                                                                 |                                                                                                         |
```
Comment (EdgarOrtegaRamirez) [inline docs/installed_plugins.md]: remove empty line
```suggestion
```

**Review by EdgarOrtegaRamirez — CHANGES_REQUESTED** [review]

**Pull request merged**